src/frontend/ts/state/chessStore.ts

```diff
@@ -220,10 +220,16 @@ const useChessStore = create<ChessState>((set, get) => ({
         var activeGames: Map<GameID, ActiveGameInfo> = get().activeGames
         activeGames.delete(gameID)
 
-        set(state => ({
-          activeGames: activeGames,
-          archivedGames: state.archivedGames.set(gameID, updatedGame)
-        }))
+        if (currentGame.isPractice === false) {
+          set(state => ({
+            activeGames: activeGames,
+            archivedGames: state.archivedGames.set(gameID, updatedGame)
+          }))
+        } else {
+          set(state => ({
+            activeGames: activeGames
+          }))
+        }
```

Comment:
Can you please comment out the lines wrapping the old `set` statement, and add a `TODO` comment about how we need a way to get practice info for games from the backend to the frontend.

`currentGame` no longer has a `practice` field after removing the commit which introduced it, meaning that we'll have to live with the bug for now.